packages/ckeditor5-block-quote/src/blockquoteediting.js

```diff
@@ -102,31 +103,49 @@ export default class BlockQuoteEditing extends Plugin {
 
 			return false;
 		} );
-	}
 
-	/**
-	 * @inheritDoc
-	 */
-	afterInit() {
-		const editor = this.editor;
-		const command = editor.commands.get( 'blockQuote' );
+		const viewDocument = this.editor.editing.view.document;
+		const selection = editor.model.document.selection;
+		const blockQuoteCommand = editor.commands.get( 'blockQuote' );
 
 		// Overwrite default Enter key behavior.
 		// If Enter key is pressed with selection collapsed in empty block inside a quote, break the quote.
-		// This listener is added in afterInit in order to register it after list's feature listener.
-		// We can't use a priority for this, because 'low' is already used by the enter feature, unless
-		// we'd use numeric priority in this case.
-		this.listenTo( this.editor.editing.view.document, 'enter', ( evt, data ) => {
-			const doc = this.editor.model.document;
-			const positionParent = doc.selection.getLastPosition().parent;
+		//
+		// Priority normal - 10 to override default handler but not list's feature listener.
+		this.listenTo( viewDocument, 'enter', ( evt, data ) => {
```

Comment:
Why did you need to change anything in the Enter key support in this PR? This PR from its history and description was all about the Backspace key, so this is confusing.

Reply:
It seems more reasonable to use `-10` instead of using `afterInit` just to order listeners.